internal/host/validator.go

```diff
@@ -1032,3 +1032,40 @@ func printIsDomainNameResolvedCorrectly(c *validationContext, status ValidationS
 		return "Unexpected status"
 	}
 }
+
+func (v *validator) isDNSWildcardNotConfigured(c *validationContext) ValidationStatus {
+	if !isNonePlatformMultiNodeInstallation(c) {
+		return ValidationSuccess
+	}
+
+	var response *models.DomainResolutionResponse
+	if err := json.Unmarshal([]byte(c.host.DomainNameResolutions), &response); err != nil {
+		return ValidationError
+	}
+	dnsWildcardName := domainNameToResolve(c, constants.DNSWildcardFalseDomainName)
+
+	for _, domain := range response.Resolutions {
+		if domain.DomainName != nil && *domain.DomainName == dnsWildcardName {
+			if len(domain.IPV4Addresses) == 0 && len(domain.IPV6Addresses) == 0 {
```

Comment:
Please add comment above this line: 
```go
// Note that we're validating that the wildcard DNS *.<cluster_name>.<base_domain> is NOT configured, because this causes known problems for OpenShift 
```

Reply:
done